test/augmentation/test_augmentation_3d.py

```diff
@@ -759,3 +760,88 @@ def test_gradcheck(self, device):
         input = torch.rand((3, 3, 3)).to(device)  # 3 x 3 x 3
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(RandomRotation3D(degrees=(15.0, 15.0)), (input, ), raise_exception=True)
+
+
+class TestRandomEqualize3D:
```

Comment:
For each test, you need to make sure that it accepts two params (device, dtype). Please also make sure your tests can run on different devices and dtypes.